src/cpp/r/R/Tools.R

```diff
@@ -1790,23 +1790,6 @@ environment(.rs.Env[[".rs.addFunction"]]) <- .rs.Env
    if (is.na(info$mode))
       return(NULL)
    
-   # Require that the netrc file be accessible only to the current user.
-   if ((info$mode & "077") != 0L)
-   {
-      warned <- getOption("rstudio.netrc.warnedAboutPermissions", default = FALSE)
-      options(rstudio.netrc.warnedAboutPermissions = TRUE)
-      
-      if (!warned)
-      {
-         fmt <- "netrc file %s has 0%s permissions; it will be ignored"
-         msg <- sprintf(fmt, shQuote(netrcPath), as.character(info$mode))
-         warning(msg, call. = FALSE)
-         warning(".netrc files should accessible to only your own user account", call. = FALSE)
-      }
-      
-      return(NULL)
-   }
-   
```

Comment:
This felt too aggressive; even curl itself doesn't require this. Surfacing this warning appropriately in all contexts was also harder than expected.